releasenotes/notes/use-abi3-4a935e0557d3833b.yaml

```diff
@@ -0,0 +1,14 @@
+---
+upgrade:
+  - |
+    By default Qiskit builds it's compiled extensions using the
+    `Python Stable ABI <https://docs.python.org/3/c-api/stable.html>`__
+    with support back to Python 3.8. This means that moving forward there
+    will be a single precompiled wheels that are shipped on release that
+    works with all of Qiskit's supported Python versions. Previously,
+    the compiled extensions were built using the version specific API and
+    would only work with a single Python version. This change was made
+    to reduce the number of package files we need to build and publish in each
+    release. When building Qiskit from source there should be no changes
+    necessary to the build process except that the default tags in the output
+    filenames will be different to reflect the use of the limited API.
```

Comment:
Might we want to change the text about "Python 3.8" to say "the oldest version of Python supported by Qiskit (currently 3.8)", or something similar?

_Maybe_ we want to flag that we don't expect any performance regressions, too?

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/10120/commits/6257bdd2e749368eba8a95f8612ed556b2c13580